pex/vendor/_vendored/wheel/wheel/metadata.py

```diff
@@ -13,7 +13,7 @@
 
 # Wheel itself is probably the only program that uses non-extras markers
 # in METADATA/PKG-INFO. Support its syntax with the extra at the end only.
-EXTRA_RE = re.compile("""^(?P<package>.*?)(;\s*(?P<condition>.*?)(extra == '(?P<extra>.*?)')?)$""")
+EXTRA_RE = re.compile(r"^(?P<package>.*?)(;\s*(?P<condition>.*?)(extra == '(?P<extra>.*?)')?)$")
```

Comment:
The vendored code should remain pristine as per: https://github.com/pantsbuild/pex/blob/master/pex/vendor/README.md

I'll follow up with a revert / re-vendor and look into adding read-only perm bits to the vendored files to make it harder to do this sort of thing when grepping for changes of a certain sort in the future.

Reply:
https://github.com/pantsbuild/pex/pull/651